src/test/java/systemtests/DeleteCommandSystemTest.java

```diff
@@ -108,8 +107,6 @@ public void delete() {
         /* Case: invalid arguments (extra argument) -> rejected */
         assertCommandFailure(DeleteCommand.COMMAND_WORD + " 1 abc", MESSAGE_INVALID_DELETE_COMMAND_FORMAT);
 
-        /* Case: mixed case command word -> rejected */
-        assertCommandFailure("DelETE 1", MESSAGE_UNKNOWN_COMMAND);
```

Comment:
Consider replacing with `assertCommandSucces()` case instead of deleting?

Reply:
Lol omg u are right hahaha